main.go

```diff
@@ -137,6 +172,20 @@ func writeOptionsFile(packageName string, node ast.Node) (found bool) {
 			}
 		}
 
+		var importList []Import
+		if imports != "" {
+			for _, s := range strings.Split(imports, ",") {
+				parts := strings.Split(s, "=")
+				if len(parts) == 1 {
+					importList = append(importList, Import{Path: parts[0]})
+				} else if len(parts) == 2 {
+					importList = append(importList, Import{Alias: parts[0], Path: parts[1]})
```

Comment:
consider checking if the second part ends with "/" + first part?  If so we wouldn't need to write out the alias?  are are you just assuming that go fmt will be run?

Reply:
We do run gofmt by default.  I think it's ok just to leave it up to the user if they want an alias or not.